app/api/threads/list_threads.feature

```diff
@@ -122,25 +122,8 @@ Feature: List threads
     When I send a GET request to "/threads?watched_group_id=@Laboratory"
     Then the response code should be 200
     And the response body should be, in JSON:
-    """
+      """
```

Comment:
(I may be wrong) does not seem aligned with the closing one

Reply:
fixed